src/App.tsx

```diff
@@ -62,7 +58,7 @@ function App () {
               fontSize={fontSize}
               fontFamily={fontFamily}
               setFontSize={setFontSize}
-              onFontChange={handleChangeTypography}
+              setFontFamily={setFontFamily}
```

Comment:
Although the additional function doesn't provide much value and I'm fine with removing it, the change to the component API (in this case, updating `FontSizeSlider` from `onFontChange` to `setFontFamily`) is, in my opinion, undesirable.

`set*` is the name given to React's state update functions. A component's API doen't need to be tied to that naming. Tomorrow, instead of updating React's local state, it may instead perform a POST http request, or use `local/sessionStorage` as the persistence mechanism.

That's where naming functions passed down to components using event-like nomenclature is helpful. The `FontSizeSlider` component notifies about a change in its state (in this case, a change in the `fontFamily`) on font change (quite literally, `onFontChange`). We discussed this on the JS lessons we do on every Monday just yestarday, in case you're interested in knowing more about the thought process: https://www.twitch.tv/videos/2040585989?t=01h11m38s.

Reply:
este comentario es CTO level, hasta timestamp de twitch tiene

Reply:
I think I've got confused by the fact that we already had `setFontSize` as props but fair point. The only issue is that we might end up with a massive `App.tsx` component but I guess that it could be refacotored into smaller components if that's the case.

Reply:
> I think I've got confused by the fact that we already had setFontSize as props

Nice call! I hadn't noticed that, it should also get refactored

To be super clear, my proposal is we get rid of the intermediate functions (as you did) but we keep the `on*` convention for the name of the props.